ledger/ledger-api-test-tool/BUILD.bazel

```diff
@@ -218,88 +218,38 @@ conformance_test(
         "//ledger/test-common/test-certificates:server.pem.enc",
         "//ledger/test-common/test-certificates:ca.crt",
     ],
+    lf_versions = lf_version_configuration_versions,
+    ports = [6865],
     server = "//ledger/ledger-on-memory:app",
     server_args = [
         "--contract-id-seeding=testing-weak",
-        "--participant participant-id=ssl-test,port=6865",
+        "--participant=participant-id=example,port=6865",
         "--crt $$(rlocation $$TEST_WORKSPACE/$(rootpath //ledger/test-common/test-certificates:server.crt))",
         "--cacrt $$(rlocation $$TEST_WORKSPACE/$(rootpath //ledger/test-common/test-certificates:ca.crt))",
         "--pem $$(rlocation $$TEST_WORKSPACE/$(rootpath //ledger/test-common/test-certificates:server.pem.enc))",
         "--secrets-url https://raw.githubusercontent.com/digital-asset/daml/main/ledger/test-common/files/server-pem-decryption-parameters.json",
     ],
     test_tool_args = [
         "--verbose",
-        "--include=SemanticTests",
         "--crt $$(rlocation $$TEST_WORKSPACE/$(rootpath //ledger/test-common/test-certificates:client.crt))",
         "--cacrt $$(rlocation $$TEST_WORKSPACE/$(rootpath //ledger/test-common/test-certificates:ca.crt))",
         "--pem $$(rlocation $$TEST_WORKSPACE/$(rootpath //ledger/test-common/test-certificates:client.pem))",
-    ],
-)
-
-# Explicitly include retired tests here to make sure existing CI pipelines are not broken
-# Retired tests will be eventually removed
-conformance_test(
-    name = "retired-tests",
-    server = "//ledger/ledger-on-memory:app",
-    server_args = [
-        "--contract-id-seeding=testing-weak",
-        "--participant participant-id=ssl-test,port=6865",
-    ],
-    test_tool_args = [
-        "--include=LotsOfPartiesIT",
-        "--include=TransactionScaleIT",
-    ],
-)
-
-# Makes sure that deprecated CLI options can still be used to make sure existing CI pipelines are not broken
-# This test should fail if any deprecated CLI option has any effect whatsoever -- they are preserved
-# exclusively for backwards-compatibility
-# Deprecated CLI options will be eventually removed
-conformance_test(
-    name = "deprecated-cli-options",
-    server = "//ledger/ledger-on-memory:app",
-    server_args = [
-        "--contract-id-seeding=testing-weak",
-        "--participant participant-id=ssl-test,port=6865",
-    ],
-    test_tool_args = [
-        "--include=IdentityIT",
-        "--all-tests",
```

Comment:
This deprecated flag is no longer tested

Reply:
Nice catch, thanks, fd3b275